docker/readme.md

```diff
@@ -0,0 +1,50 @@
+Docker Script Documentation
+===========================
+
+The files in this folder are intended to be mainly used by PyNE maintainers,
+they provide an "easy" mechanism to build and push the different containers
+required for the multiple configurations test by the PyNE CI.
+
+This folder contains:
+
+- A custom Dockerfile: "ubuntu_18.04-dev.dockerfile", allowing to build docker containers with a custom set of
+  dependencies required by the CI, the configuration of the Docker containers
+  can be specified using the "--build-args" flags. The different flags are:
```

Comment:
```suggestion
  can be specified using the `--build_***` flags. The available flags are:
```

Reply:
the correct usage in docker is actually `--build_args build_moab=YES`.

maybe `--build_args build_***=YES` is the good way to write it ?

Reply:
ah -- gotcha... that was not clear. I get it now.

If that's the case, then this place should, indeed, say [`--build-arg <flag>`](https://github.com/pyne/pyne/blob/3374b304124dd345b91c99ed3b8aa8b32fc7d85e/docker/make_pyne_docker_image.py#L66) But, the items in the list below should probably have their double dashes removed, thus:

```
- `build_moab=YES`
```